src/modules/utils.js

```diff
@@ -280,6 +280,14 @@ self = module.exports = {
             !url.href.match(/^https?:\/\/|^file:\/\//);
 
         return !!sameOrigin;
+    },
+
+    isNumber: function (value) {
+        if (typeof value === 'number') {
+            return true;
+        }
+
+        return !!value.match(/-?(\d+|\d+\.\d+|\.\d+)([eE][-+]?\d+)?/);
```

Comment:
None of the suggested methods will accept "1.2hello" as a number (that's why none of them *only* call `parseFloat()` :smile:). But yeah, you're right about the locale - none of these solutions handle that either (and it seems that is not a trivial task in JavaScript). Still, I think we're safer going with an approach like one of the above than using a regex.